css/css-animations/parsing/animation-delay-valid.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Animations Level 1: parsing animation-delay with valid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
```

Comment:
(For what it's worth, we've been dropping the author annotations in files we've been submitting since it just seems to grow over time, but if you think it's useful feel free to leave it. We've also been dropping `<html>` and `<head>` tags but that's up to you too.)